test/file05.tsx

```diff
@@ -0,0 +1,56 @@
+import { Story } from '@storybook/react';
+import { StepAnswers } from '@xsite-ui/common.questionnaire-manager';
+import { INPUT_TYPES } from '@xsite-ui/prequal-funnels.constants';
+import React, { useState } from 'react';
+
+import questionProps from '../../mocks/home-insurance-address.json';
+import Address from '../../src/address';
+import { QuestionProps } from '../../src/types';
+import { getPixelLabel } from '../../src/utils';
+import { useInvalidContext } from '../hooks/use-invalid-context';
+import { StyledCta } from '../styles';
+
+const HomeInsuranceAddress: Story<QuestionProps> = () => {
+  const { isInvalid, isInvalidTimeOut, isInvalidHandler } = useInvalidContext();
```

Comment:
Avoid using global states like this. Instead, pass the 'isInvalid', 'isInvalidTimeOut', and 'isInvalidHandler' as props or use context for a more scalable solution.